packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/ReactTextPropertiesResolver.kt

```diff
@@ -144,6 +144,12 @@ internal class ReactTextPropertiesResolver(
     }
 
     private fun getTextColor(shadowNodeWrapper: ShadowNodeWrapper): String? {
+        val isColorSet = shadowNodeWrapper
+            .getDeclaredShadowNodeField(IS_COLOR_SET_FIELD_NAME) as Boolean?
+        if (isColorSet != true) {
```

Comment:
if it is not set, won't `resolvedColor` below be null? if it is the case, we can probably simply return `#000000FF` there directly, without querying another property here

Reply:
If it is not set `resolvedColor` will actually return `0`. It will return `null` if for some reason it cannot get access to the property through reflection